ambari-server/src/main/java/org/apache/ambari/server/checks/DatabaseConsistencyCheckHelper.java

```diff
@@ -1272,7 +1270,7 @@ static void checkServiceConfigs()  {
 
         if (!MapUtils.isEmpty(configGroups)) {
           for (ConfigGroup configGroup : configGroups.values()) {
-            if (!services.containsKey(configGroup.getServiceName())) {
+            if (!Strings.isNullOrEmpty(configGroup.getServiceName()) && !services.containsKey(configGroup.getServiceName())) {
```

Comment:
IMO this should be flagged as a warning if the config group is active and has serviceName null or empty, it could have implications on other areas. cc: @mpapirkovskyy, should we also have an auto-fix for this: simplest would be to copy the tag.